03week/ticTacToe.js

```diff
@@ -35,93 +35,72 @@ function updateBoard(row, column) {
 }
 
 function isValidAnswer(row, column) {
-  const validNumber = 0 || 1 || 2 //&& board[' '] === ' ';
-  return(row === validNumber && column === validNumber) 
+  if((row == 0 || row == 1 || row  == 2) && (column == 0 || column  == 1 || column == 2)){
```

Comment:
Your check to make sure that the input is equal to 0,1,2 but your function doesn't say what to do if it doesn't match the input.